keyboards/keychron/q1/config.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2021 @ Keychron(https://www.keychron.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x3434
+#define MANUFACTURER    Keychron
+#define PRODUCT         Keychron Q1
```

Comment:
```suggestion
#define PRODUCT         Q1
```

Reply:
> May I keep the product name "Keychron Q1" ?

Reply:
Keychron is the manufacturer name, not the product name. In the QMK Toolbox it will appear as `Keychron Keychron Q1`.

Reply:
This only occur when `CONSOLE_ENABLE`  is set, and we don't really care about how QMK Toolbox show it for this is the way QMK Toolbox show the device string.  What we care is how Mac OS  & Windows show the device name, and what the user would see in device list, I think  `Keychron Q1` is easier to understand compare to `Q1`, because the os doen't concate concate the manufactor and product name.

Reply:
While I understand that, it violates the USB spec, technically.  

The USB specs have a spot for both the manufacturer and the product, and that's what QMK is setting here.  It's up to the host how that is displayed.

Reply:
The USB spec doesn't actually say very much on the topic of the manufacturer and product strings, which is probably why most OSes sorta do whatever they want with them, annoyingly.